osidb_bindings/constants.py

```diff
@@ -1,7 +1,8 @@
 from typing import Dict, List
 
 OSIDB_API_VERSION: str = "v1"
-OSIDB_BINDINGS_USERAGENT: str = "osidb-bindings-3.1.0"
+OSIDB_BINDINGS_VERSION: str = "3.1.0"
+OSIDB_BINDINGS_USERAGENT: str = f"osidb-bindings-{OSIDB_BINDINGS_VERSION}"
```

Comment:
good idea for corgi as well

Reply:
Basically I did that so I can implement later implement the version check mechanism for this [issue](https://github.com/RedHatProductSecurity/osidb-bindings/issues/11).